tests/e2e/env/bin/e2e-test-integration.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
```

Comment:
> Is it strictly necessary that we wrap the Jest process like this?

We may be able to find a way to avoid that but reiterate my comment above about smaller PRs.